docsite/docs.html

```diff
@@ -160,8 +160,37 @@ <h3 class="txt-header-2">Badge on Icon</h3>
 </div>
         </div>
 
+        <!-- Button -->
+        <div class="comp-disp" id="sec-button">
+            <h2 class="txt-header-1">Button</h2>
+          <p class="component-info txt-desc">Buttons are used for event processing and interacting with the user.Buttons are also called as call to action. </p>
+          <button class="btn-default">Button</button>
+          <button class="btn-default btn-primary-solid">Solid</button>
+          <button class="btn-default btn-primary">Primary-outline</button>
+          <button class="btn-default btn-dark">dark</button>
+          <button class="btn-default btn-light">light</button>
+          <button class="btn-default" disabled>disabled</button>
+  
+ <div class="code-snippet-display"><iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=blackboard&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%2520%2520%253Cbutton%2520class%253D%2522btn-default%2522%253EButton%253C%252Fbutton%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn-default%2520btn-primary-solid%2522%253ESolid%253C%252Fbutton%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn-default%2520btn-primary%2522%253EPrimary-outline%253C%252Fbutton%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn-default%2520btn-dark%2522%253Edark%253C%252Fbutton%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn-default%2520btn-light%2522%253Elight%253C%252Fbutton%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn-default%2522%2520disabled%253Edisabled%253C%252Fbutton%253E"
+    style="width: 730px; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+    sandbox="allow-scripts allow-same-origin">
+  </iframe>></div>
+
+        <h3 class="txt-header-2">Floating action Button</h3>
+        <a class="btn-default btn-primary btn-float-action"><i class="fas fa-plus"></i></a>

```

Comment:
add appropriate css to floating button it looks elliptical and not circular!

Reply:
updated CSS for floating button 
![Screenshot (92)](https://user-images.githubusercontent.com/73722262/152388378-577fb892-906b-4c83-aa01-22733aa4c258.png)

Reply:
updated css for floating button.